canonical-web-and-design/ubuntu.com
# Add Modal demo using react-aria

## Done

- Implemented an accessible modal using Vanilla and [react-aria](https://react-spectrum.adobe.com/react-aria/)

## QA

- Run the server with `dotrun`
- Go to http://localhost:8001/demo
- Click the button to open the modal
  - You should be able to tab through focusable elements in the modal
  - You should be able to close the modal by pressing Esc

## Screenshots

![react-aria-modal](https://user-images.githubusercontent.com/995051/148411351-520c4fad-adef-42b4-9e09-c9a53bfb1762.gif)
